assets/scss/base/_lists.scss

```diff
@@ -1,38 +1,96 @@
+/*
+Lists
 
-/**
-* @desc         lists.
-* @name         layouts
-* @author       Anthony Munene
-* @tested       N/A
-* @requires     N/A
+
```

Comment:
this block of code concerning `.list-item` shouldn't be sat at the top of this file. It should be after the 
```
ul, nav ul {
...
}
``` styles.